gcloud/orb.yaml

```diff
@@ -88,6 +88,50 @@ commands:
       - run: ln -s /root/google-cloud-sdk/bin/gsutil /usr/bin/gsutil
       - run: ln -s /root/google-cloud-sdk/bin/docker-credential-gcloud /usr/bin/docker-credential-gcloud
 
+  update-autoscaler:
+    description: >
+      Update an entry in the autoscaler dynamic configmap and reinitialize the
+      autoscaler to pick up the changes.
+    parameters:
+      config:
+        description: >
+          Local file that contains the autoscaler config for your app.
+        type: string
+      configmap:
+        description: >
+          Name of the dynamically updatable autoscaler k8s configmap.
+        default: "autoscaler-config.d"
+        type: string
+      deployment_name:
+        description: >
+          Name of the autoscaler k8s deployment.
+        default: "autoscaler"
+        type: string
+      entry_name:
+        description: >
+          Define a unique name for the configmap entry that will contain the
+          autoscaler config for your deployment, which is one of such configmap
+          entries the autoscaler will load to form its total autoscaling
+          configuration. We recommend "<your_deployment_name>.yaml".
+        type: string
+    steps:
+      - run:
+          name: construct configmap patch file
+          command: |
+            # Avoid escaping issues by using autoscaler YAML as raw (|)
+            # directly from file. Must indent raw config (using sed here).
+            echo -e "data:\n  <<parameters.entry_name>>: |\n$(sed 's/^\(.*\)/    \1/g' <<parameters.config>>)" > patch.yaml
+      - run:
+          name: update autoscaler entry in configmap
+          command: |
+            kubectl patch configmap/<<parameters.configmap>> \
+              --type merge -p "$(cat patch.yaml)"
```

Comment:
Just looking at how we do this now I noticed that we often nuke the entry explicitly before inserting the new entry like so:
```yaml
     - run:
          name: clear realtime-runner autoscaler config.d
          command: |
            kubectl patch configmap/autoscaler-config.d --type merge -p '{"data":{"realtime-nlp-runner.yaml":""}}'
```
Is it the case that this step is unnecessary?  
It makes sense it might be when I think about, just making sure we've checked this assumption.

Reply:
The step is indeed unnecessary and you can just update the configmap directly without clearing it first.